samples/accessors.cpp

```diff
@@ -27,17 +27,15 @@
  **************************************************************************/
 
 #include <CL/sycl.hpp>
-
 #include <iostream>
 
-using namespace cl::sycl;
-
 class multiply;
 
 int main() {
+  using namespace cl::sycl;  
   /* We define the data to be passed to the device. */
   int data = 5;
-
+  
```

Comment:
There's a clang-format-6 compatible config file in the SDK, would appreciate if you could run it over the files you touch